boards/arm/nucleo_g474re/nucleo_g474re_defconfig

```diff
@@ -27,16 +27,17 @@ CONFIG_CLOCK_CONTROL=y
 # Use PLLCLK for SYSCLK
 CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
 
-# Use HSI (16MHz) to feed into PLL
-CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
+# Use HSE (24MHz) to feed into PLL
+CONFIG_CLOCK_STM32_PLL_SRC_HSE=y
+CONFIG_CLOCK_STM32_HSE_CLOCK=24000000
 
 CONFIG_CLOCK_STM32_PLL_M_DIVISOR=4
```

Comment:
Proposal for an aditionnal comment to produce the PLLCLK at 170MHz, the combination would be
`CONFIG_CLOCK_STM32_PLL_M_DIVISOR=6`
with a 
`CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=85`

Reply:
Done.